docker-compose.yml

```diff
@@ -1,6 +1,16 @@
 version: "2"
+
 services:
   mongo:
-    image: mongo:3.0
+    image: mongo:3.4.3
     ports:
       - 27017:27017
+    command: --smallfiles
+  api:
+    build: .
+    depends_on:
+      - mongo
+    environment:
+      - MONGODB_URL=mongodb://mongo:27017/local?connectTimeoutMS=30000
```

Comment:
Oh right - wait - I've exposed the MongoDB port in the service above it. It's unnecessary I think, unless you want to be able to connect and have a look at it.